CMakeLists.txt

```diff
@@ -7,6 +7,37 @@ Please create a directory and run cmake from there, passing the path to this sou
 This process created the file `CMakeCache.txt' and the directory `CMakeFiles'. Please delete them.")
 endif()
 
+if(ENABLE_MASON)
+
+  # versions in use
+  set(MASON_CLANG_VERSION "3.8.1")
+  set(MASON_BOOST_VERSION "1.61.0")
+  set(MASON_STXXL_VERSION "1.4.1")
+  set(MASON_EXPAT_VERSION "2.1.1")
+  set(MASON_LUA_VERSION "5.2.4")
+  set(MASON_LUABIND_VERSION "e414c57bcb687bb3091b7c55bbff6947f052e46b")
+  set(MASON_BZIP2_VERSION "1.0.6")
+  set(MASON_TBB_VERSION "43_20150316")
+  set(MASON_CCACHE_VERSION "3.3.1")
+
+  message(STATUS "Enabling mason")
+  if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/.mason/mason.cmake")
```

Comment:
Done. Adds 3MB to third_party. Also added to the `update_dependencies.sh` script: https://github.com/Project-OSRM/osrm-backend/commit/87d09f78e6584037b6e879705e6b6a65ecabd3a8#diff-6172a0dbfdbea2ee200d1accf437674dR18